manage.py

```diff
@@ -3,7 +3,7 @@
 import sys
 
 if __name__ == "__main__":
-    os.environ.setdefault("PYTHONPATH", '/home/docker/hydroshare')
+    os.environ.setdefault("PYTHONPATH", '/hydroshare')
```

Comment:
@mjstealey May add an environment variable like ```os.environ.setdefault("TMPDIR", "/hs_tmp")``` here so that tempfile.mkdtemp() calls used in our codebase will create temp files under /hs_tmp directory rather than the default /tmp directory.